listenbrainz/listenstore/influx_listenstore.py

```diff
@@ -396,10 +406,15 @@ def dump_listens(self, location, dump_time=datetime.today(), threads=None):
                 # a temporary dir before adding them to the archive
                 for user in users:
                     username = user['musicbrainz_id']
+                    index = user['id']
                     offset = 0
 
-                    user_listens_file = '{username}.listens'.format(username=username)
-                    user_listens_path = os.path.join(listens_path, user_listens_file)
+                    directory = os.path.join(listens_path, '%03d' % (index % NUMBER_OF_USERS_PER_DIRECTORY))
```

Comment:
This will create up to 1,000 top level directories for the first 1,000 users. Also not ideal. I think the path should be: %01d/%02d/%03d , but that might be overkill....

@alastair thoughts?